sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-integration.module.ts

```diff
@@ -17,31 +12,46 @@ import { ECertPartTimeFileFooter } from "./e-cert-part-time-integration/e-cert-f
 import { ECertPartTimeFileHeader } from "./e-cert-part-time-integration/e-cert-files/e-cert-file-header";
 import { ECertPartTimeIntegrationService } from "./e-cert-part-time-integration/e-cert-part-time-integration.service";
 import { ConfigModule } from "@sims/utilities/config";
-import { SshService } from "@sims/integrations/services";
+import {
+  DisbursementScheduleErrorsService,
+  IntegrationRestrictionService,
+  SshService,
+  IntegrationStudentAssessmentService,
+} from "@sims/integrations/services";
+import { StudentService } from "@sims/services/student/student.service";
+import { SystemUsersService } from "@sims/services/system-users";
 
 @Module({
-  imports: [AuthModule, ConfigModule],
+  imports: [ConfigModule],
   providers: [
     SshService,
     ECertFullTimeIntegrationService,
     ECertPartTimeIntegrationService,
     SequenceControlService,
-    DisbursementScheduleService,
-    StudentRestrictionService,
+    DisbursementSchedulerService,
+    IntegrationStudentAssessmentService,
     ECertFileHandler,
     ECertPartTimeFileHeader,
     ECertPartTimeFileFooter,
     ECertFullTimeFileHeader,
     ECertFullTimeFileFooter,
     DisbursementScheduleErrorsService,
-    RestrictionService,
+    IntegrationRestrictionService,
     StudentService,
-    SFASIndividualService,
+    SystemUsersService,
+    StudentRestrictionsService,
   ],
   exports: [
     ECertFullTimeIntegrationService,
     ECertPartTimeIntegrationService,
     ECertFileHandler,
+    ECertPartTimeFileHeader,
+    ECertPartTimeFileFooter,
+    ECertFullTimeFileHeader,
+    ECertFullTimeFileFooter,
+    DisbursementScheduleErrorsService,
+    IntegrationRestrictionService,
```

Comment:
Apparently, there is no reason to export `IntegrationRestrictionService` here. Please ensure that only classes that are needed to be exported are exported.

Reply:
yes, we may need to revisit on the exports on the module, until its not used, we can safely remove them.